quattor/functions/validation.pan

```diff
@@ -139,3 +139,15 @@ function valid_absolute_file_paths = {
     };
     true;
 };
+
+@documentation{
+    desc = Function to validate whether the parameter received is a valid username or not; a valid username is considered to only be comprised of alpha numeric characters, plus a few special ones which are: .+:-
```

Comment:
line too long

Reply:
How do you split `desc` lines ?
Most of the examples I've seen don't split the documentation lines...

Reply:
usual `\` followed by newline works.
ideally newline and eg some indentation should work too; i opened https://github.com/quattor/pan/issues/210

Reply:
Ok! Will update. Should I use indentations or not ? (i.e. `\` then writing from the first char of the next line, or using indentations to align such as in the example you give in your opened issue ?)

Reply:
use the indentations, we'll fix the generated docs rather than make the annotation itslef unreadable/ugly